chain/vm/runtime.go

```diff
@@ -220,6 +223,23 @@ func (rt *Runtime) GetActorCodeCID(addr address.Address) (ret cid.Cid, ok bool)
 		panic(aerrors.Fatalf("failed to get actor: %s", err))
 	}
 
+	// required for genesis/testing
+	if nv := rt.NetworkVersion(); nv >= network.Version16 {
+		name, av, ok := actors.GetActorMetaByCode(act.Code)
+
+		if ok {
+			// lies, lies, lies
```

Comment:
Do we need this with the final bundle?

Reply:
We shouldn't, not anymore, thanks! Dropping!